### PR TITLE
Update campaign page

### PR DESCRIPTION
**Pull request by nkleinmann**
This PR includes work to update the campaign page based on this design: https://www.canva.com/design/DAFGOFvFl7I/xh8N29-KSkJF5EhzIQ6f6w/edit

Each main section should now be editable via Netlify.

Remaining plan to merge:
- Images, text, and links need to be checked/edited in Netlify by Erin before merging as there is lorem ipsum used

### CODE REVIEW

File: src/layouts/pages/CampaignPage.tsx [hunk 1]
```diff
@@ -6,8 +6,15 @@ import { MDXProvider } from "@mdx-js/react";
...
+import { campaign_message, campaign_problem_and_solution, campaign_priority_message, campaign_transparency_message, campaign_letter_message } from "../../../content/other/campaign_page.yml"
+import "./campaign.scss";
 
-import "./base.scss";
+// A little hacky... Ideally we should properly put this data in GQL
+const campaignMainMessage = campaign_message;
+const campaignProblemAndSolution = campaign_problem_and_solution;
+const campaignPriorityMessage = campaign_priority_message;
+const campaignTransparencyMessage = campaign_transparency_message;
+const campaignLetterMessage = campaign_letter_message;
```
Comment (fpagnoux) [inline src/layouts/pages/CampaignPage.tsx:17]: I'd maybe instead:
```ts
import { campaign } from "../../content";
const {
  campaign_message: campaignMainMessage,
  campaign_problem_and_solution: campaignProblemAndSolution,
  // ...
} = campaign;
```

**Review by fpagnoux — COMMENTED** [review]
The code looks good to me, but I think we should give different colors to different sections of the page, so that it's visually cleaner.
For instance there is a very long dark blue section starting at "Our 2023 priority".  Could we cut it in several sections of different colors?

**Comment by nkleinmann** [comment]
I'll check in with Erin about the colors!

File: src/layouts/pages/CampaignPage.tsx (hunk 1 above)
Comment (nkleinmann) [reply src/layouts/pages/CampaignPage.tsx]: Fixed here: [update imports](https://github.com/actonmass/actonmass.org/pull/223/commits/dd558d52af2d05962685f82ba22d13715867e35a)

**Review by fpagnoux — APPROVED** [review]
Good to merge once the text is updated! Thank you Nicole!

**Pull request merged**